File: warmups/05.check-if-array-is-sorted/main.js
/**
 * @param {number[]} arr
 * @param {number} n
 * @returns {boolean}
*/

class Solution {
    arraySortedOrNot(arr, n) {
        let previous = -Infinity;
        
        for (let num of arr) {
            if (previous > num) {
                return false;
            }
            previous = num;
        }
        
        return true;
    }
}

File: warmups/05.check-if-array-is-sorted/main.cpp
class Solution {
  public:
    bool arraySortedOrNot(int arr[], int n) {
        int previous = INT_MIN;
        
        for (int i = 0; i < n; i++) {
            if (previous > arr[i]) {
                return false;
            }
            
            previous = arr[i];
        }
        
        return true;
    }
};
